src/app/container/container.component.html

```diff
@@ -42,20 +42,26 @@
     </mat-card>
   </div>
 
-  <div class="row m-1">
+  <div class="row m-1" fxLayout.lt-sm="column">
     <ngx-json-ld [json]="schema"></ngx-json-ld>
-    <div class="col-md-7">
-      <h3 id="tool-path">
-        <span id="verifiedIcon" *ngIf="(extendedTool$ | async)?.versionVerified">
-          <a [href]="getVerifiedLink()" class="verified-check">
-            <mat-icon matTooltip="Verified">done</mat-icon>
-          </a>
+    <div fxFlex="80">
+      <h3 id="tool-path" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="1rem">
+        <span>
+          <img src="../../assets/svg/tool-circle.svg" alt="tool icon" />
         </span>
         <span *ngIf="tool?.private_access" class="private-lock">
           <app-private-icon></app-private-icon>
         </span>
         {{ tool?.tool_path }}<span *ngIf="tool?.workflowVersions.length > 0">:</span
-        ><span class="ds-green">{{ selectedVersion?.name }}</span>
+        ><span class="ds-steelblue">{{ selectedVersion?.name }}</span>
+        <span id="verifiedIcon" *ngIf="(extendedTool$ | async)?.versionVerified" fxHide.lt-sm>
+          <mat-chip matTooltip="Verified" class="container-label-chip">
+            <a [href]="getVerifiedLink()" class="verified-check" fxLayoutGap="0.5rem" fxLayoutAlign=" center">
```

Comment:
How often is this invoked? It's generally not good to invoke functions like this from HTML to set an attribute value, as they get invoked a lot.

Reply:
was like that before. the function does get called a lot and should definitely be changed